spec/api-browser-window-spec.ts

```diff
@@ -5410,7 +5410,7 @@ describe('BrowserWindow module', () => {
     });
 
     ifdescribe(process.platform !== 'win32')('visibleOnAllWorkspaces state', () => {
-      describe('with properties', () => {
+      it('with properties', () => {
         it('can be changed', () => {
```

Comment:
I thought nested `it` doesn't work? Also looks like this change isn't part of the original PR that this PR is backporting

Reply:
Good catch, I'm not sure what went wrong there. :thinking: 

Fixed with force-push cd1c7fcf19